terraform/modules/happy-service-eks/variables.tf

```diff
@@ -114,7 +114,7 @@ variable "platform_architecture" {
 
 variable "aws_iam_policy_json" {
   type        = string
-  default     = ""
+  default     = "{}"
```

Comment:
I think this might also be invalid from my memory. Try something like the following:

https://stackoverflow.com/questions/62287436/is-it-possible-to-have-a-no-op-iam-policy

Reply:
Yep, as-is this errors out:
```
Error: updating IAM policy arn:aws:iam::xxxxx:policy/mysandbox-gql-dev-mysandbox: MalformedPolicyDocument: Syntax errors in policy.
```

Reply:
Maybe instead we can always create the aws_iam_role, but create the aws_iam_role_policy and aws_iam_role_policy_attachment resources conditionally

Reply:
yeah, I have a branch that implements that here: https://github.com/chanzuckerberg/happy/compare/jgadling/targetfixes?expand=1